conda-recipe/meta.yaml

```diff
@@ -25,7 +25,8 @@ requirements:
     - python
     - setuptools
     - numba   {{ NUMBA_VERSION }}
-    - numpy
+    - numpy                           # [win or osx or py==38]
```

Comment:
@tomashek, @xaleryb I would prefer not to submit this as there's no requirement for SDC package itself to be built with specific numpy version. Also there are no description in the commit that explains why this change is needed and when it can be reverted (if it's some kind of workaround, at least some GitHub reference is needed). Also there should generally be more proper ways of fixing numpy version during specific conda builds (e.g. CONDA_NPY env variable may be set).